triton/apps/python-conda.rst

```diff
@@ -0,0 +1,461 @@
+==============================
+Python environments with Conda
+==============================
+
+Conda is a popular package manager that is especially
+popular in data science and machine learning communities.
+
+It is commonly used to handle complex requirements of Python
+and R packages.
+
+Quick usage guide
+*****************
+
+First time setup
+----------------
+
+You can get conda by loading the ``miniconda``-module:
+
+.. code-block:: bash
+
+  module load miniconda
+
+By default Conda stores installed packages and environments in your home
+directory. However, as home directory has a lower quota, it is a good idea
+to tell conda to install packages and environments into your work directory:
+
+.. code-block:: bash
+
+  mkdir $WRKDIR/.conda_pkgs
+  mkdir $WRKDIR/.conda_envs
+
+  conda config --append pkgs_dirs ~/.conda/pkgs
+  conda config --append envs_dirs ~/.conda/envs
+  conda config --prepend pkgs_dirs $WRKDIR/.conda_pkgs
+  conda config --prepend envs_dirs $WRKDIR/.conda_envs
+
+Now you're all set up to create your first environment.
+
+
+Creating a simple environment with conda
+----------------------------------------
+
+One can install environments from the command line itself, but a better idea
+is to write an ``environment.yml``-file that describes the environment.
+
+Below we have a simple ``environment.yml``:
+
+.. code-block:: yaml
+
+  name: conda-example
+  channels:
+    - conda-forge
+  dependencies:
+    - blas * mkl
```

Comment:
blas seems like something beyond what most peolpe would use.  what about an example of pinning a ML library to a specific version as the starting example?

Reply:
@tpfau had the same comment. Fixed it in the new commit. MKL & BLAS is mentioned later in the complex environment section.